### PR TITLE
feat[wip]: add initial trezor structure in pali

### PR DESCRIPTION
**Pull request by lucasgabrielgsp**
- [x] add initial trezor structure in pali

P.s.: For test this PR, use feat/add-trezor branch in sysweb3 repository

### CODE REVIEW

File: source/components/Header/AccountHeader.tsx [hunk 1]
```diff
@@ -76,13 +78,60 @@ const RenderAccountsListByBitcoinBased = (
...
-            ))
-        : Object.entries(accounts).map(
-            ([keyringAccountType, accountTypeAccounts]) => (
-              <div key={keyringAccountType}>
-                {Object.values(accountTypeAccounts)
-                  .filter((account) => account.xpub !== '')
-                  .map((account, index, { length }) => (
+            ))}
+
+          {activeNetwork.currency !== 'tsys' &&
```
Comment (claudiovb) [inline source/components/Header/AccountHeader.tsx:83]: Does trezor works on testnet from evm ?

Comment (claudiovb) [reply source/components/Header/AccountHeader.tsx:83]: We probably should block it from any utxo testnet that the user might add not only tsys, by checking through the keyringManaget istestnet function that should be straightforward

File: source/components/Header/AccountHeader.tsx [hunk 2]
```diff
@@ -76,13 +78,60 @@ const RenderAccountsListByBitcoinBased = (
...
+          ([keyringAccountType, accountTypeAccounts]) => (
+            <div key={keyringAccountType}>
+              {Object.values(accountTypeAccounts)
+                .filter((account) => account.xpub !== '')
+                .map((account, index, { length }) => {
+                  if (account.isTrezorWallet) return null;
+                  return (
                     <Tooltip
                       key={account.id}
                       childrenClassName={`${index === 0 && 'mt-1'} flex w-full`}
```
Comment (claudiovb) [inline source/components/Header/AccountHeader.tsx:137]: Having a tooltip for trezor account would be nice

File: source/pages/Settings/ConnectHardwareWallet.tsx [hunk 3]
```diff
@@ -11,20 +11,38 @@ import { getController } from 'utils/browser';
...
+  const trezorAccounts = Object.values(accounts.Trezor);
 
-  const handleCreateHardwareWallet = async () => {
-    await controller.wallet.account.sys.trezor.createAccount();
+  const getSlip44 = () => {
+    let slip44 = '';
+
+    switch (isBitcoinBased) {
+      case true:
+        slip44 = '57';
```
Comment (claudiovb) [inline source/pages/Settings/ConnectHardwareWallet.tsx:25]: This is only for syscoin we probably should add the blockchain id by the connected chain ID and exclude utxo testnets in the isBitcoinBased case

**Review by claudiovb — CHANGES_REQUESTED** [review]

File: source/components/Header/AccountHeader.tsx (hunk 1 above)
Comment (lucasgabrielgsp) [reply source/components/Header/AccountHeader.tsx:83]: > Does trezor works on testnet from evm ?

Yep, it do.

**Review by claudiovb — APPROVED** [review]

**Pull request merged**